charts/hoprd-operator/values.yaml

```
@@ -201,7 +198,10 @@ operator:
       ## @param operator.fastSync.source.deployment Name of the hoprd node deployment use as source for Fast Sync
       ## Required: true when fast-sync is enabled
       deployment: ""
-    
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)